__tests__/feature/searchAndOpenTemplate.test.js

```diff
@@ -0,0 +1,76 @@
+import { renderApp, createHistory, createStore } from 'testUtils'
+import { act, fireEvent, screen } from '@testing-library/react'
+import * as sinopiaSearch from 'sinopiaSearch'
+import Config from 'Config'
+
+// This forces Sinopia server to use fixtures
+jest.spyOn(Config, 'useResourceTemplateFixtures', 'get').mockReturnValue(true)
+jest.mock('sinopiaSearch')
+
+// Mock jquery
```

Comment:
I don't think you need to mock jquery (I commented out this line and the test ran fine)